wolfcrypt/src/port/nxp/dcp_port.c

```diff
@@ -386,6 +386,11 @@ int wc_Sha256Final(wc_Sha256* sha256, byte* hash)
     return ret;
 }
 
+int wc_Sha256FinalRaw(wc_Sha256* sha256, byte* hash)
+{
+    return wc_Sha256Final(sha256, hash);
```

Comment:
Final raw is not meant to add padding and length at the end.

Reply:
Thanks @SparkiDev . What do you suggest in this case? The DCP does not expose such an interface.

Reply:
Why do you need wc_Sha256FinalRaw()?
Most of the time callers don't need it or have alternative code.

Reply:
It looks like TLS code is pulling that in via HMAC. Please see the ZD ticket.